protos/feast/core/FeatureService.proto

```diff
@@ -46,3 +50,34 @@ message FeatureServiceMeta {
   google.protobuf.Timestamp last_updated_timestamp = 2;
 
 }
+
+
+message LoggingConfig {
+  float sample_rate = 1;
+  google.protobuf.Duration partition_interval = 2;
+
+  oneof destination {
+    FileDestination file_destination = 3;
+    BigQueryDestination bigquery_destination = 4;
+    RedshiftDestination redshift_destination = 5;
+    SnowflakeDestination snowflake_destination = 6;
+  }
```

Comment:
Is there a goal of making feature logging extensible? How will third-party offline stores enable feature logging?

Suggestion: we could have destination be a map<string, string> (essentially a flag JSON). We're losing strong typing, but other offline stores can also use it.

Reply:
That's done similarly to [data source proto](https://github.com/feast-dev/feast/blob/master/protos/feast/core/DataSource.proto), so the plan is to add CustomDestination with map or bytes later.

Reply:
Added custom destination proto message.

Reply:
Added custom destination proto message.